pkg/trtl/config/config_test.go

```diff
@@ -23,10 +25,20 @@ var testEnv = map[string]string{
 	"TRTL_REPLICA_REGION":           "us-east-1c",
 	"TRTL_REPLICA_GOSSIP_INTERVAL":  "30m",
 	"TRTL_REPLICA_GOSSIP_SIGMA":     "3m",
-	"TRTL_METRICS_ADDR":             ":7777",
-	"TRTL_METRICS_ENABLED":          "true",
+	"TRTL_INSECURE":                 "true",
 	"TRTL_MTLS_CHAIN_PATH":          "fixtures/certs/chain.pem",
 	"TRTL_MTLS_CERT_PATH":           "fixtures/certs/cert.pem",
+	"TRTL_BACKUP_ENABLED":           "true",
+	"TRTL_BACKUP_INTERVAL":          "1h",
+	"TRTL_BACKUP_STORAGE":           "fixtures/backups",
+	"TRTL_BACKUP_KEEP":              "7",
+}
+
+var strategyEnv = map[string]string{
+	"TRTL_REPLICA_STRATEGY_HOSTNAME_PID": "true",
+	"TRTL_REPLICA_HOSTNAME":              "pizza-36",
+	"TRTL_REPLICA_STRATEGY_FILE_PID":     "",
+	"TRTL_REPLICA_STRATEGY_JSON_CONFIG":  "testdata/replicas.json",
```

Comment:
Separated this from the main tests since it's just a k8s thing. Everything should work without this.